prime-router/src/main/kotlin/fhirengine/azure/FHIRFunctions.kt

```diff
@@ -127,7 +128,13 @@ class FHIRFunctions(
         fhirEngine: FHIRTranslator,
         actionHistory: ActionHistory = ActionHistory(TaskAction.translate),
     ) {
-        runFhirEngine(message, dequeueCount, fhirEngine, actionHistory)
+        val messagesToDispatch = runFhirEngine(message, dequeueCount, fhirEngine, actionHistory)
```

Comment:
Can we add a comment that this only dispatches an event if the message is going through the send original logic?